src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements.mdx

```diff
@@ -0,0 +1,1093 @@
+---
+title: '.NET agent: compatibility and requirements'
+tags:
+  - Agents
+  - NET agent
+  - Getting started
+metaDescription: "APM's .NET agent for .NET Framework applications: compatibility and requirement, and what frameworks are automatically instrumented."
+redirects:
+  - /docs/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework
+  - /docs/agents/net-agent/getting-started/compatibility-requirements-net-agent
+  - /docs/agents/net-agent/getting-started/compatibility-requirements-net-framework-agent
+  - /docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core/#requirements
+  - /docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework/#requirements
+freshnessValidatedDate: never
+---
+
+New Relic's .NET agent supports both .NET Framework and .NET Core. This document describes compatibility and support for .NET Core applications. 
+
+The agent includes built-in instrumentation for some of the most popular parts of the .NET ecosystem, including frameworks, databases, and message queuing systems. After you [download and install](/install/dotnet) the agent, it runs within the monitored process. The agent doesn't create a separate process or service.
```

Comment:
Example comment: I think that these sentences should be merged. Example: Blha blah